mne/viz/_figure.py

```diff
@@ -1,333 +1,60 @@
-        selections_dict = self.mne.parent_fig.mne.ch_selections
-        buttons = self.mne.radio_ax.buttons
-        labels = [label.get_text() for label in buttons.labels]
-        this_label = buttons.value_selected
-        parent = self.mne.parent_fig
-        if this_label == 'Custom' and not len(selections_dict['Custom']):
-            with _events_off(buttons):
-                buttons.set_active(self.mne.old_selection)
-            return
-        # clicking a selection cancels butterfly mode
-        if parent.mne.butterfly:
-            parent._toggle_butterfly()
-            with _events_off(buttons):
-                buttons.set_active(labels.index(this_label))
-        parent._update_selection()
-
-    def _set_custom_selection(self):
-        """Set custom selection by lasso selector."""
-        chs = self.lasso.selection
-        parent = self.mne.parent_fig
-        buttons = self.mne.radio_ax.buttons
-        if not len(chs):
-            return
-        labels = [label.get_text() for label in buttons.labels]
-        inds = np.in1d(parent.mne.ch_names, chs)
-        parent.mne.ch_selections['Custom'] = inds.nonzero()[0]
-        buttons.set_active(labels.index('Custom'))
-
-    def _style_radio_buttons_butterfly(self):
-        """Handle RadioButton state for keyboard interactions."""
-        # Show all radio buttons as selected when in butterfly mode
-        parent = self.mne.parent_fig
-        buttons = self.mne.radio_ax.buttons
-        color = (buttons.activecolor if parent.mne.butterfly else
-                 parent.mne.bgcolor)
-        for circle in buttons.circles:
-            circle.set_facecolor(color)
-        # when leaving butterfly mode, make most-recently-used selection active
-        if not parent.mne.butterfly:
-            with _events_off(buttons):
-                buttons.set_active(self.mne.old_selection)
-        # update the sensors too
-        parent._update_highlighted_sensors()
-
-
-class MNEBrowseFigure(MNEFigure):
-    """Interactive figure with scrollbars, for data browsing."""
-
-    def __init__(self, inst, figsize, ica=None, xlabel='Time (s)', **kwargs):
-        from matplotlib.colors import to_rgba_array
-        from matplotlib.ticker import (FixedLocator, FixedFormatter,
-                                       FuncFormatter, NullFormatter)
-        from matplotlib.patches import Rectangle
-        from matplotlib.widgets import Button
-        from matplotlib.transforms import blended_transform_factory
-        from mpl_toolkits.axes_grid1.axes_size import Fixed
-        from mpl_toolkits.axes_grid1.axes_divider import make_axes_locatable
+    def __init__(self, **kwargs):
         from .. import BaseEpochs
         from ..io import BaseRaw
         from ..preprocessing import ICA
 
-        super().__init__(figsize=figsize, inst=inst, ica=ica, **kwargs)
+        self._data = None
+        self._times = None
+
+        self.mne = BrowserParams(**kwargs)
```

Comment:
This is a convention introduced in the matplotlib-backend to access parameters and gui-elements across parent and childs or depending gui-objects. I adapted it (before I had something similar with passing the parent-class onto the childs). The matplotlib-backend relies havily on this convetion so changing something there would have big consequences.
I think the name is just short which is good since it is used very frequently.